### ISSUE TITLE
TypeError: Cannot read property 'collection' of undefined

### ISSUE THREAD

**Issue by pauldariye** [codesandbox/codesandbox-client#2777]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System |
  | ------- | --------- | ---------------- |
  | chrome | 79.0.3941 | OS X 10.15.0           |

  **Route:**
  https://codesandbox.io/dashboard/sandboxes/devtools-workshop

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'collection' of undefined
  TypeError: Cannot read property 'collection' of undefined
    at https://codesandbox.io/static/js/app.7b7fc602.js:438:224745
    at https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:11000
    at https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:355473
    at f.trackScope (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1468208)
    at o.wrappedComponent (https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:355442)
    at Li (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2052774)
    at ps (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2059858)
    at Ka (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2097141)
    at Fa (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2081589)
    at Na (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2078609)
    at Ra (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2077169)
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2039070
    at t.unstable_runWithPriority (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2113916)
    at co (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2038799)
    at po (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2039016)
    at ho (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2038951)
    at Oa (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2076160)
    at Object.enqueueSetState (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2043805)
    at r.w.setState (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2335969)
    at https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:354880
    at e.callback (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1467884)
    at h.flush (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1470729)
    at d.flush (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1467029)
    at Object.cb (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1355213)
    at e.emit (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1351588)
    at internal.setJwt (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1357380)
    at Proxy.<anonymous> (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:1356480)
    at https://codesandbox.io/static/js/default~app~embed.959d3548.chunk.js:276:242742
    at a (https://codesandbox.io/static/js/common-sandbox.f352477c.chunk.js:1:14585)
    at Generator._invoke (https://codesandbox.io/static/js/common-sandbox.f352477c.chunk.js:1:14373)
    at Generator.e.<computed> [as next] (https://codesandbox.io/static/js/common-sandbox.f352477c.chunk.js:1:15007)
    at t (https://codesandbox.io/static/js/common-sandbox.f352477c.chunk.js:1:8537)
    at u (https://codesandbox.io/static/js/common-sandbox.f352477c.chunk.js:1:8747)
  
    in Unknown
    in r
    in t
    in Xf
    in t
    in t
    in Unknown
    in t
    in withRouter()
    in div
    in t
    in elements__Content
    in div
    in div
    in t
    in elements__Container
    in n
    in r
    in t
    in withRouter(r)
    in t
    in t
    in div
    in t
    in elements__Content
    in n
    in t
    in withRouter(n)
    in div
    in t
    in elements__Container
    in Unknown
    in t
    in withRouter()
    in DragDropContext(withRouter())
    in t
    in t
    in l
    in t
  ```

  </details>

**Comment by lbogdan** [comment]
Looks like `data.me` is `undefined` here:
https://github.com/codesandbox/codesandbox-client/blob/729d375557db6d21a692184e91bae14c12116df2/packages/app/src/app/pages/Dashboard/Content/routes/PathedSandboxes/index.js#L36

**Comment by hetpatel33** [comment]
@Saeris @CompuIves @lbogdan 
Can someone please review my fix for this? 
PR is #2882

**Comment by SaraVieira** [comment]
Closed by #2225 2882
